html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <h1>Coding Form!</h1>
+  </head>
+  <body>
+    <div></div>
+    <label for="email">Email</label>
+    <div></div>
+    <input type="email" />
+    <div></div>
+    <label for="why-coding">Why is Coding Cool?</label>
+    <div></div>
+    <select id="custom-form">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+    <div></div>
+    Favorite Language (choose one)
+    <div></div>
+    <input type="radio" name="favoriteLanguage" value="html" />HTML
+    <input type="radio" name="favoriteLanguage" value="CSS" />CSS
+    <input type="radio" name="favoriteLanguage" value="JavaScript" />JavaScript
+    <div></div>
+    <label for="checkbox"></label>
+    Would you like to sign up for more information?
+    <div></div>
+    <input type="checkbox" name="Sign me up" value="Sign me up" />Sign me up!
+    <div></div>
+    <form action="http://index.html">Comments</form>
```

Comment:
The `form` element should wrap your entire form's content, this looks like you are using the `form` element as a `label` 😬